src/main/java/appeng/util/item/AEItemStack.java

```diff
@@ -108,7 +108,19 @@ public static AEItemStack fromPacket(final PacketBuffer buffer) {
         final boolean isCraftable = buffer.readBoolean();
         final long stackSize = buffer.readVarLong();
         final long countRequestable = buffer.readVarLong();
-        final ItemStack itemstack = buffer.readItemStack();
+
+        // based on buffer.readItemStack()
+        // Adapted to also handle forge capabilities
+        ItemStack itemstack = ItemStack.EMPTY;
+        if (buffer.readBoolean()) {
+            int i = buffer.readVarInt();
+            int j = buffer.readByte();
```

Comment:
i, j. come on! :-P

Reply:
So we can just copy&paste it once forge changes something.